src/javascript/JContent/ContentTree/ContentTree.jsx

```diff
@@ -17,7 +18,7 @@ export const ContentTree = ({lang, siteKey, path, openPaths, setPath, openPath,
         openPaths.push(rootPath);
     }
 
-    const {treeEntries, refetch} = useTreeEntries({
+    const json = {
```

Comment:
I would probably call it something other than json :) but other than that looks fine

Reply:
The universal json variable is all I can think about haha
Maybe `useTreeOptionsJson` is better?

Reply:
Anything that would tie it to the fact that it is used in the useTreeEntries hook is better for me )

Reply:
okie sounds good